packages/tooling/vis/src/commands/run.ts

```
@@ -244,13 +245,30 @@ const run: Command = {
+        // workspace root. Other fields keep their existing spread semantics
+        // to avoid changing override precedence for parallel/dryRun/etc.
+        const configTaskRunnerOptions
+            = (config.taskRunnerOptions ?? {}) as TaskRunnerOptions & { cacheDirectory?: string };
+        const resolvedCacheDirectory = resolveCacheDirectory(
+            workspaceRoot,
+            options.cacheDir as string | undefined,
+            configTaskRunnerOptions.cacheDirectory,
+        );
+
         const runnerOptions: TaskRunnerOptions = {
             cacheDirectory: options.cacheDir as string | undefined,
```

Biome (2.4.10)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)